test/operator/installation.go

```diff
@@ -25,6 +31,40 @@ func InstallFromSource(withCsi bool) features.Func {
 	return manifests.InstallFromFiles(paths)
 }
 
+func InstallViaMake() features.Func {
+	return func(ctx context.Context, t *testing.T, environmentConfig *envconf.Config) context.Context {
+		rootDir := getCorrectDir()
+
+		platform := kubeobjects.ResolvePlatformFromEnv()
+		makeTarget := "deploy"
+		if platform == kubeobjects.Openshift {
+			makeTarget = strings.Join([]string{makeTarget, "openshift"}, "/")
+		} else if platform == kubeobjects.Kubernetes {
+			makeTarget = strings.Join([]string{makeTarget, "kubernetes"}, "/")
+		}
+
+		err := exec.Command("make", "-C", rootDir, makeTarget).Run()
+		if err != nil {
+			t.Fatal("failed to install the operator via the make command")
+			return nil
+		}
+
+		return ctx
+	}
+}
+
+func getCorrectDir() string {
```

Comment:
What qualifies a directory to be "correct"?
```suggestion
func getSourceRootDir() string {
```